XMTPiOSExample/XMTPiOSExample/Views/ConversationListView.swift

```diff
@@ -66,10 +69,29 @@ struct ConversationListView: View {
 			await MainActor.run {
 				self.conversations = conversations
 			}
+
+			await add(conversations: conversations)
 		} catch {
 			print("Error loading conversations: \(error)")
 		}
 	}
+
+	func add(conversations: [Conversation]) async {
+		// Ensure we're subscribed to push notifications on these conversations
+		do {
+			try await XMTPPush.shared.subscribe(topics: conversations.map(\.topic))
```

Comment:
For a full featured push notification example, we are going to have to subscribe to more than just conversation topics.

We are going to want to subscribe to the `invite` and `intro` topics to trigger a background update to the conversation list when a new invite/intro is received.

Reply:
That's a good call, I should add it to this PR.